src/rgw/rgw_admin.cc

```diff
@@ -8355,7 +8355,14 @@ int main(int argc, const char **argv)
 
     formatter->open_object_section("attrs");
     for (iter = other_attrs.begin(); iter != other_attrs.end(); ++iter) {
-      dump_string(iter->first.c_str(), iter->second, formatter.get());
+      bufferlist& bl = iter->second;
+      if (iter->first == RGW_ATTR_OBJ_REPLICATION_TIMESTAMP) {
+        decode_dump<ceph::real_time>("user.rgw.replicated-at", bl, formatter.get());
+      } else if (iter->first == RGW_ATTR_OBJ_REPLICATION_TRACE) {
+        decode_dump<uint32_t>("user.rgw.replication-trace", bl, formatter.get());
```

Comment:
decode_dump() is just calling encode_json(), and src/common/ceph_json.h has an overload for ceph::real_time

so `decode_dump<ceph::real_time>()` should display this as a date string instead of an integer

Reply:
leaving it as a string for now.